### ISSUE TITLE
WPEM plugin accessibility

### ISSUE THREAD

**Issue by ashokdudhat** [wpeventmanager/wp-event-manager#908]
Hi WPEM team!

I wanted to provide your team with some quick jumping off points to optimize the WP Event Manager plugin for accessibility! The following are a few examples of things you could easily implement for a better universal front-end experience:
Proper heading hierarchy: Particular in your single template, the headings (h2, h3, h4, etc), should be properly nested per WCAG 2.0 Heading standards. This will not only help with accessibility, but make your plugin more SEO-friendly (which appeals to a huge audience)
:focus indication: Anywhere there is a :hover pseudo-class effect, there should also be a :focus effect (links, buttons, etc) per WCAG 2.0 guidelines. :hover will only change activity indication for mouse users, while :focus benefits keyboard users. Example: a {color: red; text-decoration: underline;} a:hover, a:focus {color: black; text-decoration: none;}
Color Contrast: Your default #00a5fa blue for links doesn't meet minimum contrast requirement (you can use WebAim's Contrast Checker). Meaning out of the box, your links are inaccessible and can't be seen by people with low vision due to a low 2.7:1 contrast ratio on a white background (must be at least a 4.5:1 ratio to pass).

These are just a few examples of more simple changes you could implement to increase your accessibility. As we've become quick fans of the plugin, we'd be happy to discuss performing a full audit of your plugin. The Event plugin market is in desperate need of a user-friendly, attractive, AND accessible option which just isn't available right now -- however, WPEM seems to be the closest without even trying which tells me you have an excellent staff of developers.

If you're interested in an audit, please reply all to this email and Bet Hannon (who is CC'd) will be in touch.

https://bhmbizsites.com/accessibility-tips-for-wp-event-manager/
